ProxyAppsSuite/tests/examini.yaml

```
@@ -0,0 +1,103 @@
+
+examini:
+   summary: Research vehicle for particule codes in Molecular Dynamics which allows modular independent investigation of different aspects such as force calculation, communication, neighbor list construction and binning.
+  
+
+   build:
+       modules: [ gcc, cmake, openmpi/2.1.2 ]
+       source_path: ExaMini
+       env:
+          CC: mpicc
+       cmds:
+           # Building KOKKOS
+           - git clone https://github.com/kokkos/kokkos ~/kokkos
```

```suggestion
           - git clone https://github.com/kokkos/kokkos 
```
We want Pavilion to `git clone` kokkos in the current directory it's in (within the builds directory) and not necessarily in our home directory (which is where `~/kokkos` is)